packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -270,14 +270,15 @@ export default class CallDiagnosticMetrics {
   /**
    * Prepare the event and send the request to metrics-a service.
    * @param event
+   * @returns promise
    */
-  submitToCallDiagnostics(event: Event) {
+  submitToCallDiagnostics(event: Event): Promise<any> {
```

Comment:
if you do not care what is return value, please use `Promise<void>`

Reply:
fixed